release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1038,6 +1044,8 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* In the Red Hat OpenStack Platform ({rh-openstack}), a known potential issue in {rh-openstack} Kuryr may occur when attempting to create `300` ports in bulks of `10` ports and `10` subnets. These bulk requests can create conflicts when distributing IP addresses. This issue was resolved in a previous errata; however, Red Hat suggests setting a value lower than `300` to avoid this issue in future releases. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2024690[*BZ2024690*])
```

Comment:
It might be better not to mention these numbers because it's very likely that is possible to hit with other values.

Reply:
@MaysaMacedo That makes complete sense! I have updated the PR to reflect the suggested changes! :)